rke/k8s_rke_system_images.go

```diff
@@ -5779,10 +5778,10 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/kubelet-pause:v0.1.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.15.13",
 		},
-		// Enabled out of band post v2.5.9
-		"v1.20.10-rancher1-1": {
+		// Enabled in v2.6.1
```

Comment:
`Enabled in v2.6.1` should be for `v1.20.11-rancher1-2` and this would be `Enabled out of band post v2.5.x`

Reply:
Due to the OOB release, this comment will need to be `Enabled out of band post v2.5.9`